src/readwrite.rs

```diff
@@ -163,6 +163,7 @@ impl<'env> Iterator for Iter<'env> {
 }
 
 /// Wrapper around an `lmdb::Database`.
+#[derive(Copy, Clone)]
```

Comment:
I think it would be worth commenting that at this time, the underlying LMDB handle is a C integer, and therefore `Copy` is automatic :)

Reply:
@ncalexan Noted in 5ae8508!